sdk/core/azure-core/test/nlohmann-json-test/src/unit-class_parser.cpp

```diff
@@ -0,0 +1,2126 @@
+/*
```

Comment:
Since this file isn't included in the CMakeList.txt, let's remove it from the check-in/git history.